wallet-api/src/Wallet/Emulator/Http.hs

```diff
@@ -41,8 +51,18 @@ type WalletAPI
      :<|> "wallets" :> Capture "walletid" Wallet :> "transactions" :> ReqBody '[ JSON] Tx :> Post '[ JSON] ()
      :<|> "wallets" :> "transactions" :> Get '[ JSON] [Tx]
 
+type ControlAPI
+   = "blockchain" :> "actions" :> Get '[ JSON] [Tx]
```

Comment:
ok. it came from an effort to be RESTful to some degree, since it returns a list of the transactions that have been carried out successfully it's sort of the latest transactions of the blockchain object. But basically this is pointless effort.

Reply:
Yeah let's now prematurely generalise here.